submissions/dummy_submission/the-nuclear-options.md

```diff
@@ -0,0 +1,150 @@
+
+In the early game when U-235 is slow to obtain, steam storage prevents fuel waste: When your power demand is low, excess steam builds up in the system. If the steam system is full, the boilers stop and the heat starts building up in the reactors, which provide an additional energy buffer up until 1000°C. After that, the reactors continue to accept and burn fuel cells no matter what, wasting them. A steam storage setup prevents this scenario, and it is even more effective if you configure your uranium fuel cell inserters to activate only when the steam storage is running low (you can do this with a single red/green cable between each inserter and a steam storage tank). However, after unlocking the Kovarex Enrichment Process, uranium fuel cells become very easy to obtain and so letting the reactors heat up and dump heat is not a big concern any longer. However, you might still want to keep the steam storage after Kovarex if you want to keep your reactors below 900°C for safety reasons.
+
+Another reason to store steam is to make a "steam battery" as type of power buffer: When the nuclear plant's energy demand is below 100% of what reactors supply in the form of heat, the steam buffer is filled. Then, if your steam production gets disrupted by a biter bursting a water pipe or a brief reactor fuel shortage, the buffer can keep your turbines running for a few additional seconds or minutes. Meanwhile, if you build extra steam turbines (or use the generous ratio of two turbines per heat exchanger) your steam buffer can be used on top of your 100% reactor heat supply in situations where there is a temporary high power demand, like a biter attack on laser turrets. Note that the same power buffering goals can be achieved also using accumulators, which are far better for UPS than turbines. Also note that every steam turbine added to a power grid is listed as extra capacity in the power overview menu regardless of its steam supply. This means that if you have an excess of turbines and an emptied steam buffer, you will be unable to supply as much power as listed in the overview even though your reactors are running at full capacity.
+
+#### Making your plant pretty *and* practical
+
+Having covered the scientific aspects of nuclear plant design, now we can consider the artistic aspects. While it may seem like there is not much room for creativity because of all the constraints mentioned above, you do have a lot of freedom about how you arrange your components. The biggest setback here is the neighbor bonus, which makes it difficult to prefer any setup other than 2xN. However, if you are willing to sacrifice some neighbor bonus efficiency (especially when fuel is abundant after unlocking Kovarex) you can get creative and arrange your reactors in rings or fractals.
```

Comment:
```suggestion
Having covered the scientific aspects of nuclear plant design, we can now consider the artistic aspects. While it may seem like there is not much room for creativity because of all the constraints mentioned above, you do have a lot of freedom about how you arrange your components. The biggest setback here is the neighbor bonus, which makes it difficult to prefer any setup other than 2xN. However, if you are willing to sacrifice some neighbor bonus efficiency (especially when fuel is abundant after unlocking Kovarex) you can get creative and arrange your reactors in rings or fractals.
```